packages/util/form-utils/package.json

```diff
@@ -0,0 +1,76 @@
+{
+    "name": "@navikt/k9-form-utils",
+    "version": "0.0.1",
+    "scripts": {
+        "build": "rollup -c rollup.config.js",
+        "clean": "rm -rf ./dist ./node_modules",
+        "storybook": "start-storybook -p 7001 --ci",
```

Comment:
Sett gjerne port til 7003 og legg til en ny i `refs`-lista i `.storybook/main.js` på rot, dette for at det skal bli med i kjøringen av Storybook Composition :)